script.js

```diff
@@ -1,4 +1,165 @@
+//Requirements
+//There are 2 players and players take turns.
+//When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+//The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+//After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+//1) a) create a dice game with 2 dices for player 1, b) let player 1 to choose which dice they want to be the first one.
+//2) include player 2 and let them do same things as player 1
+//3)compare the 2 dice rolls and announce the winner
+
+//Global variables:
+var GAME_STATE_ROLL_DICE = "GAME_STATE_ROLL_DICE ";
+var GAME_STATE_CHOOSE_DICE_ORDER = "GAME_STATE_CHOOSE_DICE_ORDER";
+var GAME_STATE_COMPARE_SCORES = "GAME_STATE_COMPARE_SCORES";
+var GAME_STATE_PLAY_AGAIN = "GAME_STATE_PLAY_AGAIN";
```

Comment:
good use of string constants! the value itself does not need to be upper cased unlike the variable naming since it is afterall, a string.